server/core/gw_utils.c

```diff
@@ -61,6 +61,12 @@ setipaddress(struct in_addr *a, char *p)
 
     hint.ai_socktype = SOCK_STREAM;
 
+    if (strchr(p, '%') != NULL)
+    {
+        MXS_ERROR("Host %s contains wildcard, return", p);
```

Comment:
Tested with:

user@ foo%bar.bar
user@%.foo.bar

The patch can be applied but log message should be INFO:

MXS_ERROR ===> MXS_INFO